nomenclature/testing.py

```diff
@@ -36,11 +36,14 @@ def assert_valid_structure(path: Path):
         :class:`DataStructureDefinition`
       - If a `mappings` folder exists, it must be a valid :class:`RegionProcessor`
```

Comment:
Since there's two new arguments for the function a docstring explaining what they do would be great.
Since the current doc string is not in the standard format, I could go ahead and re-write it to numpydoc style if that's ok for you.

Reply:
OK for me. I can also try to do it if you prefer.

Reply:
Thanks, would be great if you could do that.